client/src/app/site/site.component.ts

```diff
@@ -121,11 +102,14 @@ export class SiteComponent extends BaseComponent implements OnInit {
         }
     }
 
+    // TODO
```

Comment:
please add a proper comment here.
Necessary for a decent todo-list

(TODO: Add editProfile functionality & TODO: changePassword functionality)